src/api.js

```diff
@@ -123,3 +124,25 @@ export async function getSingleMovieReview(
     return reviewArray;
   }
 }
+
+// data = array of screenings, today = todays time and date in ISO format
+export function filterOutOldScreenings(data, today){
+ 
+  // run through the array and return objects containing time >= todaysTime
+  const res = data.filter(comming);
+    
+  // currentValue is reqired parameter but not needed, index is index of current object
+      function comming(currentValue, index){
```

Comment:
This is a filter function. It should return `true` or `false` to indicate whether an entry in the array it's used upon should be included in the output array or not.